src/main/scala/org/camunda/feel/impl/builtin/ListBuiltinFunctions.scala

```diff
@@ -43,9 +43,9 @@ object ListBuiltinFunctions {
     "distinct values" -> List(distinctValuesFunction),
     "flatten" -> List(flattenFunction),
     "sort" -> List(sortFunction),
-    "joining" -> List(joiningFunction,
-                      joiningWithDelimiterFunction,
-                      joiningWithDelimiterAndPrefixAndSuffixFunction)
+    "join" -> List(joinFunction,
```

Comment:
:x: We must rename the function to `string join`. In the spec, the function is defined as `string join()`. It is a new practice in the spec to use the data type as prefix :sweat_smile: 

```suggestion
    "string join" -> List(joinFunction,
```

Reply:
oh, sorry, I missed that! I am going to fix it.